docs/governance/decred-constitution.md

```diff
@@ -1,11 +1,8 @@
 # <img class="dcr-icon" src="/img/dcr-icons/Constitution.svg" /> Decred Constitution
 
-!!! note
-    The constitution was written prior to Decred's launch; some portions of it are no longer relevant and some are yet to be realized. Politeia will be used, when it launches, to propose amendments to the constitution for ratification by Voters.
+*Decred* (/ˈdi:ˈkred/, /dɪˈkred/, dee-cred) is a secure, adaptable and self-funding cryptocurrency with a system of community-based governance integrated into its blockchain. The project mission is to develop technology for the public benefit, with a primary focus on cryptocurrency technology. This constitution defines a set of principles which guide the decision-making of the project's stakeholders, and describes the processes through which the blockchain and Treasury are governed. The purpose of this document is to explain what the Decred currency and project is about, so that interested parties can make a more informed judgment about whether it is something they are into. This document is not intended to take precedence over the Decred network's consensus rules, should any conflict arise.
```

Comment:
> The purpose of this document is to explain what the Decred currency and project is about, so that interested parties can make a more informed judgment about whether it is something they are into.

This sentence reads poorly in comparison to the others in this paragraph.  Perhaps something like "This document is meant to manage the expectations of prospective and actual Decred users, so that they more clearly understand the social contract between themselves and the project."